### PR TITLE
create frontend part 3

### PR DESCRIPTION
**Pull request by at-ngatran**
Hello a Dung, I fixed branch, please check and merge it, thanks

### CODE REVIEW

File: public/frontend/js/main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+jQuery(function($) {'use strict',
+
+	//#main-slider
+	$(function(){
+		$('#main-slider.carousel').carousel({
+			interval: 8000
```
Comment (dzungnguyenh) [inline public/frontend/js/main.js]: [rank C] What does '8000' mean? You need to define it as constant.

File: public/frontend/js/main.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		event.preventDefault();
+		var form_status = $('<div class="form_status"></div>');
+		$.ajax({
+			url: $(this).attr('action'),
+
+			beforeSend: function(){
+				form.prepend( form_status.html('<p><i class="fa fa-spinner fa-spin"></i> Email is sending...</p>').fadeIn() );
+			}
+		}).done(function(data){
+			form_status.html('<p class="text-success">' + data.message + '</p>').delay(3000).fadeOut();
```
Comment (dzungnguyenh) [inline public/frontend/js/main.js]: [rank C] What does 'delay(3000)' mean? You need to define it as constant or global variable.

File: public/frontend/js/main.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		});
+	});
+
+	
+	//goto top
+	$('.gototop').click(function(event) {
+		event.preventDefault();
+		$('html, body').animate({
+			scrollTop: $("body").offset().top
+		}, 500);
```
Comment (dzungnguyenh) [inline public/frontend/js/main.js]: [rank C] What does '500' mean? You need to define it as constant.

**Comment by at-ngatt** [comment]
hi a Dung, I fixed difine, please check it, thanks

**Comment by at-ngatran** [comment]
hi a Dung, I fixed define, please check it, thanks

**Pull request merged**